keyboards/handwired/dactyl_manuform_pi_pico/config.h

```diff
@@ -0,0 +1,10 @@
+// Copyright 2023 Gustaw.xyz (@Gustaw.xyz)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#pragma once
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET
+#define RP2040_BOOTLOADER_DOUBLE_TAP_RESET_TIMEOUT 200U
+#define SERIAL_USART_FULL_DUPLEX
+#define SERIAL_USART_TX_PIN GP9
+#define SERIAL_USART_RX_PIN GP8
+#define EE_HANDS
```

Comment:
Not a huge fan of eehands on an ARM board. It's ... more complicated.  If you have any GPIO pins left, the split pin handedness is a much better option.

Reply:
Thank you for the suggestions. I switched to #define SPLIT_HAND_PIN GP22, soldered the jumpers and it looks like it works well.